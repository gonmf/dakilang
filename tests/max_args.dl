# Assuming current maximum of 20 arguments

a(X) :- b(A00, A01, A02, A03, A04, A05, A06, A07, A08, A09, A10, A11, A12, A13, A14, A15, A16, A17, A18, A19).

c(X) :- b(A00, A01, A02, A03, A04, A05, A06, A07, A08, A09, A10, A11, A12, A13, A14, A15, A16, A17, A18, A19, A20).
