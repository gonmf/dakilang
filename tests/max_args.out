> # Assuming current maximum of 20 arguments
>
> a(X) :- b(A00, A01, A02, A03, A04, A05, A06, A07, A08, A09, A10, A11, A12, A13, A14, A15, A16, A17, A18, A19).
>
> c(X) :- b(A00, A01, A02, A03, A04, A05, A06, A07, A08, A09, A10, A11, A12, A13, A14, A15, A16, A17, A18, A19, A20).
Syntax error at c(X) :- b(A00, A01, A02, A03, A04, A05, A06, A07, A08, A09, A10, A11, A12, A13, A14, A15, A16, A17, A18, A19, A20).: Too many arguments in a single clause (max is 20): this is an interpreter-specific setting
